test/compositemark/errorbar.test.ts

```diff
@@ -668,8 +668,8 @@ describe('normalizeErrorBar with aggregated upper and lower bound input', () =>
             y2: {field: 'upper_people', type: 'quantitative'},
             x: {field: 'age', type: 'ordinal'},
             tooltip: [
-              {field: 'upper_people', title: 'Upper error of people', type: 'quantitative'},
-              {field: 'lower_people', title: 'Lower error of people', type: 'quantitative'},
+              {field: 'lower_people', title: 'people', type: 'quantitative'},
```

Comment:
People and people2 is worse than upper/lower error

Reply:
This is ranged error bar case, should we use the original field name?

Reply:
The axis title should just be the field name (e.g., `people`). 

However, the title in the tooltip should be quite descriptive. 
(What does the error exactly represent? -- ci0/ci1, mean+stdev/err?)